web/static/js/components/surveys/SurveyWizardRespondentsStep.jsx

```diff
@@ -59,22 +59,31 @@ class SurveyWizardRespondentsStep extends Component {
     if (data.invalidEntries.length == 0) {
       content = (
         <div className='card-content card-error'>
-          <div><b>{t('The file you uploaded does not contain any phone number')}</b></div>
+          <div><b>{t('The file you uploaded does not contain any phone number or respondent id')}</b></div>
           <div>{t('Please upload antother file.')}</div>
         </div>
       )
     } else {
-      const lineNumbers = data.invalidEntries.slice(0, 3).map((entry) => entry.line_number)
+      const invalidPhoneNumberEntries = data.invalidEntries.filter(invalidEntry => invalidEntry.type != 'not-found')
```

Comment:
given a list of entries, we can extract a function `{count, extraCount, lineNumbers} = reportError(invalidEntries)` to make the following lines more readable.

Reply:
What is the type for invalid phone number? i don't see it.

I suggest using a more explicit value like `invalid-phone-number`, `invalid-respondent-id`. It will be easier to track in the source code.